confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConf.java

```diff
@@ -97,6 +97,8 @@ public class ConfigNodeConf {
   /** The number of DataRegions of each StorageGroup */
   private int dataRegionCount = 1;
 
+  private String adminName = "root";
```

Comment:
We have this in IoTDBConfig, this could be moved to node-commons

Reply:
ok

Reply:
ok